# frame names
frame_ids: [local_origin]
use_ocam: false

############### Camera parameters (for top-mounted Mobius cam) ###############
camera_delay: 130 # ms

camera_offset_x: 0.0
camera_offset_y: 0.24
camera_offset_z: -0.11

camera_offset_roll: 90.0
camera_offset_pitch: -120.0
camera_offset_yaw: 0.0

# Simulation parameters
# camera_offset_x: 0.0
# camera_offset_y: -0.04
# camera_offset_z: -0.2

# camera_offset_roll: 45 #degrees
# camera_offset_pitch: -90  #degrees
# camera_offset_yaw: 45  #degrees

############### Distance estimation parameters ###############
dist_filter_window: 3
max_dist: 15.0
xy_covariance_coeff: 0.001
z_covariance_coeff: 0.005
dist_corr_p0: -1.0
dist_corr_p1: 0.7
